src/docs/DescriptionFromComment.stories.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts" context="module">
+	import { Story, Template } from '@storybook/addon-svelte-csf';
+	import type { Meta } from '@storybook/svelte';
+
+	import DescriptionComponent from './DescriptionComponent.svelte';
+
+	/**
+	 * This description was set above the `export const meta` inside the module tag `<script context="module">`.
+	 */
+	export const meta = {
+		title: 'Description From Comment/Svelte CSF',
```

Comment:
Could you move all stories under a single "Description" title? Like `"Description/From Comment/Svelte CSF"`. I think that would help a bit with navigating the Storybook